src/aktualizr_repo/repo_test.cc

```diff
@@ -123,6 +123,7 @@ TEST(aktualizr_repo, delegation) {
   EXPECT_EQ(targets["signed"]["delegations"]["roles"][0]["name"].asString(), "test_delegate");
   EXPECT_EQ(targets["signed"]["delegations"]["roles"][0]["paths"][0].asString(), "tests/test_data/*.txt");
 
+  std::cout << "KEYTYPE: " << keytype_stream.str() << "\n";
```

Comment:
Did you mean to leave this line in? It looks like this commit was duplicated during a merge or rebase, but this version of it only serves to add this one line. Ideally this bogus commit should probably be eliminated.